test/unit/services/test_item.py

```diff
@@ -3,1304 +3,884 @@
+                                    will be added automatically.
+        :param catalogue_category_in_data: Either `None` or a dictionary containing the catalogue category data as would
+                                           be required for a `CatalogueCategoryIn` database model.
+        :param usage_status_in_data: Dictionary containing the basic usage status data as would be required for a
+                                     `UsageStatusIn` database model.
+        """
+
+        # Generate mandatory IDs to be inserted where needed
+        catalogue_item_id = str(ObjectId())
+        system_id = str(ObjectId())
+        usage_status_id = str(ObjectId())
+
+        ids_to_insert = {
+            "catalogue_item_id": catalogue_item_id,
+            "system_id": system_id,
+            "usage_status_id": usage_status_id,
+        }
+
+        # Catalogue category
+        catalogue_category_in = None
+        if catalogue_category_in_data:
+            catalogue_category_in = CatalogueCategoryIn(**catalogue_category_in_data)
+
+        catalogue_category_id = str(ObjectId())
+        self._catalogue_category_out = (
+            CatalogueCategoryOut(
                 **{
-                    **ITEM_INFO,
-                    "properties": add_ids_to_properties(None, ITEM_INFO["properties"]),
+                    **catalogue_category_in.model_dump(by_alias=True),
+                    "_id": catalogue_category_id,
                 },
             )
+            if catalogue_category_in
+            else None
         )
-    catalogue_item_repository_mock.get.assert_called_once_with(catalogue_item_id)
-    item_repository_mock.create.assert_not_called()
-    assert str(exc.value) == f"No catalogue item found with ID: {catalogue_item_id}"
-
-
-def test_create_with_non_existent_usage_status_id(
-    test_helpers, item_repository_mock, usage_status_repository_mock, item_service
-):
-    """
-    Test creating an item with a non-existent usage status ID.
-    """
-    usage_status_id = str(ObjectId())
-
-    # Mock `get` to not return a catalogue item
-    test_helpers.mock_get(usage_status_repository_mock, None)
-
-    with pytest.raises(MissingRecordError) as exc:
-        item_service.create(
-            ItemPostSchema(
-                catalogue_item_id=str(ObjectId()),
-                usage_status_id=usage_status_id,
-                system_id=str(ObjectId()),
-                **{
-                    **ITEM_INFO,
-                    "properties": add_ids_to_properties(None, ITEM_INFO["properties"]),
-                },
+        ServiceTestHelpers.mock_get(self.mock_catalogue_category_repository, self._catalogue_category_out)
+
+        # Catalogue item
+
+        # When properties are given need to add any property `id`s and ensure the expected data inserts them as well
+        catalogue_item_property_post_schemas = []
```

Comment:
```suggestion
```

Reply:
Its used below.

Reply:
Sorry I should have put a note. It is used but only inside that `IF` statement. What you currently have on line 157 should be just enough.

Reply:
Oh right, yeah no sorry I didn't realise 😄